pkg/dhclient/dhclient.go

```diff
@@ -27,7 +27,7 @@ import (
 	"golang.org/x/sys/unix"
 )
 
-const (
+var (
```

Comment:
I suspect that if you write a test that changes this, and tests run in parallel, you will trigger the race detector, it would be good to know either way.